backend/cn/symRel.mli

```diff
@@ -1,89 +0,0 @@
-(* Module SymPair -- TODO: BCP: What is it for?? *)
-
-(* TODO: BCP: Some filenames use_underscores, while some areInCamlCase
-   -- any compelling reason for that? *)
```

Comment:
None at all. Originally everything (in CN at least) was CamlCase, then other people added different names.